docs/UserGuide.md

```diff
@@ -81,74 +81,66 @@ Format: `add-patient n/NAME p/PHONE e/EMAIL a/ADDRESS [t/TAG]…​`
 </div><br>
 
 Examples:
-
 * `add-patient n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01`
 
-### Cleaning all entries in patient records: `clear-patient`
+### Clearing all entries in patient records: `clear-patient`
 Clears all entries from the patient records. <br>
-If any appointments exist in the appointment schedule, then this command will not work. 
-`clear-appt` will have to be done first to clear the appointment schedule.
-
+* Entries in the patient records can only be cleared if there are no existing appointments in the appointment schedule. <br>
+* Otherwise, `clear-appt` have to called first. <br>
 
 Format: `clear-patient`
 
 ### Deleting a patient : `delete-patient`
-Deletes the specified patient from the patient records.
+Deletes the specified patient from the patient records.<br>
 
-Format: `delete-patient INDEX`
+Format: `delete-patient [--force] INDEX`
```

Comment:
related to the earlier comment: is there a need to specify explicitly that the `--force` must be before `INDEX`?